docs/self-managed/setup/upgrade.md

```diff
@@ -2,130 +2,61 @@
-export OPTIMIZE_SECRET=$(kubectl get secret "camunda-optimize-identity-secret" -o jsonpath="{.data.optimize-secret}" | base64 --decode)
-export OPERATE_SECRET=$(kubectl get secret "camunda-operate-identity-secret" -o jsonpath="{.data.operate-secret}" | base64 --decode)
-export CONNECTORS_SECRET=$(kubectl get secret "camunda-connectors-identity-secret" -o jsonpath="{.data.connectors-secret}" | base64 --decode)
-export ZEEBE_SECRET=$(kubectl get secret "camunda-zeebe-identity-secret" -o jsonpath="{.data.zeebe-secret}" | base64 --decode)
-export KEYCLOAK_ADMIN_SECRET=$(kubectl get secret "camunda-keycloak" -o jsonpath="{.data.admin-password}" | base64 --decode)
-export KEYCLOAK_MANAGEMENT_SECRET=$(kubectl get secret "camunda-keycloak" -o jsonpath="{.data.management-password}" | base64 --decode)
-export POSTGRESQL_SECRET=$(kubectl get secret "camunda-postgresql" -o jsonpath="{.data.postgres-password}" | base64 --decode)
-```
-
-After exporting all secrets into environment variables, run the following upgrade command:
-
-```shell
-helm upgrade camunda camunda/camunda-platform \
-  # Uncomment if Console is enabled.
-  # --set global.identity.auth.console.existingSecret=$CONSOLE_SECRET \
-  --set global.identity.auth.tasklist.existingSecret=$TASKLIST_SECRET \
-  --set global.identity.auth.optimize.existingSecret=$OPTIMIZE_SECRET \
-  --set global.identity.auth.operate.existingSecret=$OPERATE_SECRET \
-  --set global.identity.auth.connectors.existingSecret=$CONNECTORS_SECRET \
-  --set global.identity.auth.zeebe.existingSecret=$ZEEBE_SECRET \
-  --set identityKeycloak.auth.adminPassword=$KEYCLOAK_ADMIN_SECRET \
-  --set identityKeycloak.auth.managementPassword=$KEYCLOAK_MANAGEMENT_SECRET \
-  --set identityKeycloak.postgresql.auth.password=$POSTGRESQL_SECRET
+helm search repo camunda/camunda-platform --versions
 ```
 
-:::note
+### Helm CLI version
 
-If you have specified on the first installation certain values, you have to specify them again on the upgrade either via `--set` or the values file and the `-f` flag.
+Use the recommended Helm CLI version for your Helm chart when upgrading. The Helm CLI version for each chart can be found on the [chart version matrix](https://helm.camunda.io/camunda-platform/version-matrix/).
 
-:::
+## Update your configuration
 
-For more details on the Keycloak upgrade path, you can also read the [Keycloak Upgrading Guide](https://www.keycloak.org/docs/latest/upgrading/).
+Configuration adjustments may be required when upgrading to a new version of the Helm chart. Before beginning your upgrade, ensure you have implemented any changes required by your new version.
 
-## Version update instructions
+<Tabs groupId="upgrades" defaultValue="8.5" queryString values={
+[
+{label: 'From Camunda 8.4 to 8.5', value: '8.5', },
+{label: 'From Camunda 8.3 to 8.4', value: '8.4', },
+{label: 'From Camunda 8.2 to 8.3', value: '8.3', },
```

Comment:
Since this is the /next/ version, you would want to include 8.6, right? Or is this just reorganizing the current content only?

Reply:
Reorg of the current only - I don't think we have any notes yet for what config changes will be required in 8.5 > 8.6.